### ISSUE TITLE
Water arrow don't like the nether

### ISSUE THREAD

**Issue by TPC25** [An-Sar/PrimalCore#258]
Got a crash while in the nether shooting a water arrow. Was just testing around in there and shooting water arrow and hitting mobs is fine but if you shoot and hit a block for the arrow to stick, that's when it crash.

Am using Build 88, not ready to go to 90 as I don't know if this is fix in that build. I saw build 90 has fixes about worldgen and mud but I haven't had any problem with that so I'm comfortable at build 88.

https://pastebin.com/M5uYxQsi

**Comment by An-Sar** [comment]
yeah the mud crash is extremely rare, and doesn't persist after a reboot which has made it hard to resolve

this has been fixed (0.6.91)
